lib/components/Input/CheckboxInput.tsx

```diff
@@ -52,6 +54,14 @@ export interface CheckboxInputProps extends React.Props<CheckboxInputType> {
     attr?: CheckboxInputAttributes;
 }
 
+const StyledActiveCheckboxButton = styled(Attr.span)`
+    &&&& {
+        color: ${(props: ThemeProps<ShellTheme>) => props.theme && props.theme.colorPrimaryButtonTextRest };
```

Comment:
in which case would props.theme be not defined

Reply:
Yes, this check is not necessary, theme will always be an object.